docker/versions.sh

```diff
@@ -13,41 +13,13 @@ export PIKA_VERSION=1.1.0
 export REDIS_VERSION=3.5.3
 
 # set all the git tags or tickets for the versions you want to use for the containers
-export DM_CSC_BASE_GIT=tags/4.2.0
-#
-export DM_ATARCHIVER_GIT=tags/4.1.0
-#
-export CTRL_OODS_GIT=tags/5.0.1
-#
-export DM_CCARCHIVER_GIT=tags/3.1.0
-#
-export DM_CONFIG_CATCHUP_GIT=master
-export DM_CATCHUPARCHIVER_GIT=master
-#
-export LSST_STACK_VERSION=7-stack-lsst_distrib-w_2022_06
-export LSSTTS_DEPLOY_ENV_VERSION=${CYCLE_TAG}
+export CTRL_OODS_GIT=tickets/DM-24127
```

Comment:
Not quite sure, but I suspect you may need to change tag/branch before merging the ticket branch to main.

Reply:
Yes, that was my plan.  Thanks for the reminder though